pennylane/io.py

```diff
@@ -410,6 +410,62 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
+def from_qiskit_noise(noise_model, **kwargs):
+    """Converts a Qiskit `NoiseModel <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.noise.NoiseModel.html>`__
+    into a PennyLane :class:`~.NoiseModel`.
+
+    .. note::
+
+        This function depends upon the PennyLane-Qiskit plugin. Follow the
+        `installation instructions <https://docs.pennylane.ai/projects/qiskit/en/latest/installation.html>`__
+        to get up and running. You may need to restart your kernel if you are running in a notebook
+        environment.
+
+    Args:
+        noise_model (qiskit_aer.noise.NoiseModel): a ``NoiseModel`` constructed in Qiskit
+        kwargs: Optional keyword arguments for conversion of the noise model.
+
+    Keyword Arguments:
+        kraus_shape (bool): use shape of the Kraus operators to display ``qml.QubitChannel``
+            instead of the complete list of matrices. Default is ``True``.
+        decimal_places (int): number of decimal places to round the Kraus matrices. Default is ``10``.
+
+    Returns:
+        qml.NoiseModel: An equivalent noise model constructed in PennyLane
+
+    Raises:
+        ValueError: When an encountered quantum error cannoted be converted.
+
+    .. note::
+
+        Currently, PennyLane noise models does not support readout errors, so those will be skipped during conversion.
+
+    **Example**
+
+    Consider the following noise model constructed in Qiskit:
+
+    >>> import qiskit.providers.aer.noise as noise
+    >>> error_1 = noise.depolarizing_error(0.001, 1) # 1-qubit noise
+    >>> error_2 = noise.depolarizing_error(0.01, 2) # 2-qubit noise
+    >>> noise_model = noise.NoiseModel()
+    >>> noise_model.add_all_qubit_quantum_error(error_1, ['rz', 'ry'])
+    >>> noise_model.add_all_qubit_quantum_error(error_2, ['cx'])
+
+    This noise model can be converted into PennyLane using:
+
+    >>> qml.from_qiskit_noise(noise_model)
+    NoiseModel({
+        OpIn(['RZ', 'RY']): QubitChannel(Klist=Tensor(4, 4, 4))
+        OpIn(['CNOT']): QubitChannel(Klist=Tensor(16, 4, 4))
+    })
+    """
+    try:
+        plugin_converter = plugin_converters["qiskit_noise"].load()
```

Comment:
I tried running the code example here and I got this error 🤔

```
>>> qml.from_qiskit_noise(noise_model)
Traceback (most recent call last):
  File "/Users/isaac/Documents/pennylane/pennylane/io.py", line 463, in from_qiskit_noise
    plugin_converter = plugin_converters["qiskit_noise"].load()
                       ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
KeyError: 'qiskit_noise'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/isaac/Documents/pennylane/pennylane/io.py", line 466, in from_qiskit_noise
    raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
RuntimeError: Conversion from Qiskit requires the PennyLane-Qiskit plugin. You can install the plugin by running: pip install pennylane-qiskit. You may need to restart your kernel or environment after installation. If you have any difficulties, you can reach out on the PennyLane forum at https://discuss.pennylane.ai/c/pennylane-plugins/pennylane-qiskit/
```

Reply:
@isaacdevlugt you still have this warning with the PL-Qiskit warning installed?

Reply:
I think you need to re-install the PL-Qiskit plugin to register the new entry point.

Reply:
Yes works for me now! Thanks guys :)